lib/hammer_cli/output/definition.rb

```diff
@@ -12,10 +12,117 @@ def append(fields)
       @fields += fields
     end
 
+    def delete_fields!(by = {})
+      return clear if by.empty?
+      return deep_delete!(@fields, by) if by[:deep_delete]
+      ids, labels = adjust_options(by)
+      @fields.delete_if do |f|
+        label = f.label
+        label = f.label.downcase unless label.nil?
+        ids.include?(f.id) || labels.include?(label)
+      end
+    end
+
+    def select_fields!(by = {})
+      return self if by.empty?
+      return @fields = deep_select(@fields, by) if by[:deep_select]
+      ids, labels = adjust_options(by)
+      @fields.keep_if do |f|
+        label = f.label
+        label = f.label.downcase unless label.nil?
+        ids.include?(f.id) || labels.include?(label)
+      end
+    end
+
+    def replace_field(field_id, fields = nil, &block)
```

Comment:
This method is handy! I like using the block.
Does it also accept a single field? Using is as `replace_field(:name, Field.new(:key => :surname))` would be nice too.